intelmq/bots/BOTS

```diff
@@ -655,6 +655,11 @@
                 "filter_value": "<PT>"
             }
         },
+        "Fqdn2Ip": {
```

Comment:
How about renameing this to ``gethostbyname`` ?
This way it would be clear (IMHO - but that can be discussed) by its name already what it does.